frontend/src/app/AboutDialog.tsx

```diff
@@ -81,6 +117,75 @@ const AboutDialog: React.FC<AboutDialogProps> = ({ onClose }) => {
                 : 'Unknown'}
             </Content>
           </Content>
+
+          {/* Component Releases Section */}
+          <Content
+            style={{
+              paddingTop: 48,
+              visibility: loading ? 'hidden' : 'visible',
+            }}
+          >
+            <h4>Installed components</h4>
+            {groupedComponents.length > 0 ? (
+              <>
+                <Content>
+                  <Table
+                    aria-label="Component Releases Table"
+                    data-testid="component-releases-table"
+                  >
+                    <Thead>
+                      <Tr data-testid="table-row-title">
+                        <Th width={10}>
+                          {isRHOAI
+                            ? RhoaiDefaultComponentReleaseName
+                            : OdhDefaultComponentReleaseName}{' '}
+                          Component
+                        </Th>
+                        <Th width={20}>Upstream component</Th>
+                        <Th width={10}>Upstream version</Th>
+                      </Tr>
+                    </Thead>
+                    <Tbody>
+                      {groupedComponents.map(([displayName, details]) =>
+                        details.releases.length > 0 ? (
+                          details.releases.map((release, index) => (
+                            <Tr key={`${displayName}-${index}`} data-testid="table-row-data">
+                              {index === 0 ? (
+                                <Td rowSpan={details.releases!.length}>{displayName}</Td>
+                              ) : null}
+                              <Td className="hover-cell" style={{ paddingInlineStart: 16 }}>
+                                {release.repoUrl ? (
+                                  <a
+                                    href={release.repoUrl}
+                                    target="_blank"
+                                    rel="noopener noreferrer"
+                                  >
+                                    {release.name} <ExternalLinkAltIcon style={{ marginLeft: 8 }} />
+                                  </a>
+                                ) : (
+                                  release.name
+                                )}
+                              </Td>
+                              <Td>{release.version}</Td>
+                            </Tr>
+                          ))
+                        ) : (
+                          <Tr key={displayName}>
+                            <Td>{displayName}</Td>
+                            <Td colSpan={3} style={{ fontStyle: 'italic', color: 'gray' }}>
```

Comment:
Is italic grey a style requested by UX?
At the very least, we need to use PF variables for colors.

Otherwise just render plain text to keep it simple.

Reply:
No, the UX doesn't cover this specific case. I'm happy with the plain text.

Reply:
@kywalker-rh can you please check this? 
This is what it looks like without italic grey:

![Screenshot 2025-02-27 at 12 59 31](https://github.com/user-attachments/assets/3970ce43-7fe2-4503-bbdb-57a160c23bdb)

Reply:
I'm removing the italic gray style as well, it looks good without it.